linode/manager
# feat: [M3-6146] - Add User Data to Linode Rebuild flow

## Description 📝
Adds User Data section to the Linode Rebuild flow when an image with `cloud-init` capability is selected.

## Preview 📷
![Screenshot 2023-03-07 at 11 33 22 AM](https://user-images.githubusercontent.com/114682940/223487141-234304c0-911c-4a2c-ae13-f70efe967406.jpg)

## To-Do

## How to test 🧪
- Ensure that the existing User Data work elsewhere in the app has not been adversely impacted by changes made on this branch
- Confirm that Linode rebuild requests go through with the `{ metadata: { user_data: [encoded string ...] } }` property (or `null` instead of encoded string when the Reuse checkbox is marked)
